src/include/test/unit_queries.cc

```diff
@@ -113,6 +113,7 @@ TEST_CASE("test queries", "[queries]") {
       }
     }
 
+#ifdef TILEDB_VS_ENABLE_BLAS
```

Comment:
Why do we need this? Wasn't the test passing before without BLAS?

Reply:
x-ref https://app.shortcut.com/tiledb-inc/story/40724

Reply:
Don't quite follow, doesn't that issue suggest we should not be adding more things referring to BLAS? Or are you saying that `gemm_query()` only works with BLAS?